### PR TITLE
Tiny fix of README

### PR DESCRIPTION
**Pull request by tianxin1860**
### PR types

Others
### PR changes

Models
### Description

1. Tiny fix of README.md

### CODE REVIEW

File: examples/text_matching/ernie_matching/README.md [hunk 1]
```diff
@@ -61,10 +61,10 @@ python -u -m paddle.distributed.launch --gpus "0" train_pointwise.py \
 model = ppnlp.transformers.ErnieGramModel.from_pretrained('ernie-gram-zh')
 tokenizer = ppnlp.transformers.ErnieGramTokenizer.from_pretrained('ernie-gram-zh')
 
-# 使用ernie预训练模型
-# ernie
-#model = ppnlp.transformers.ErnieModel.from_pretrained('ernie'))
-#tokenizer = ppnlp.transformers.ErnieTokenizer.from_pretrained('ernie')
+# 使用 ernie 预训练模型
```
Comment (ZeyuChen) [inline examples/text_matching/ernie_matching/README.md]: ernie -> ERNIE

Comment (ZeyuChen) [reply examples/text_matching/ernie_matching/README.md]: 区分代码用语和书面用语

**Review by ZeyuChen — DISMISSED** [review]
LGTM

File: examples/text_matching/ernie_matching/README.md (hunk 1 above)
Comment (tianxin1860) [reply examples/text_matching/ernie_matching/README.md]: 已修正，thanks

**Review by yingyibiao — APPROVED** [review]

**Pull request merged**